tests/osproc/texitsignal.nim

```diff
@@ -36,4 +36,4 @@ if paramCount() == 0:
     echo p.waitForExit() == 128 + SIGKILL
 
 else:
-  sleep(5000)  # should get killed before this
+  os.sleep(5000)  # should get killed before this
```

Comment:
why was this change needed?

Reply:
Test was using the wrong `sleep`, because both `os` and `posix` have it, fully qualified is as explicit as it gets,
whatsoever the test is not about sleeps https://github.com/nim-lang/Nim/pull/17149/commits/4464d04dc8aceed03f674df30ba114a24a82c1b4

Reply:
If the change breaks our very own test suite, it'll probably also break somebody else's code. Sorry, but this has to be rejected because of this.

Reply:
Documentation explicitly says to use fully qualified names when code is ambiguous. :shrug: